### ISSUE TITLE
Feed does not work on Podcasts for iOS

### ISSUE THREAD

**Issue by hammady** [nerab/dropcaster#15]
Please look at this feed:
https://s3-us-west-2.amazonaws.com/hammadypublic/majalis-baqara/main.rss
Although all links are valid and can be played from the browser, but the stock Podcasts app in iOS 10 shows no episodes.

**Comment by nerab** [comment]
I am getting this error:

```
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>7999C22243D3BE0D</RequestId>
<HostId>
+lQt2gTRQR456BC+8IKURej0zjd7Qd8w6l2DpP0kofBVTSR2rtxFm+nVS5CF1z64qN/VeLrytBU=
</HostId>
</Error>
```

Has the link expired? Maybe you can refresh it or attach the source to this issue?

**Comment by hammady** [comment]
The correct link is: 
https://s3-us-west-2.amazonaws.com/hammadypublic/majalis-baqara/index.rss
It was fixed already by #16.

**Comment by nerab** [comment]
Nice. Thanks for the follow-up!
